engine/src/main/java/org/destinationsol/game/ObjectManager.java

```diff
@@ -44,12 +45,13 @@
     private final World myWorld;
     private final Box2DDebugRenderer myDr;
     private final HashMap<SolObject, Float> myRadii;
+    private final Context myContext;
```

Comment:
We're trying to phase out all the `myX` variables so let's just call this `context` please.